pom.xml

```diff
@@ -1041,7 +1041,7 @@
               </ignoredUsedUndeclaredDependencies>
               <ignoredUnusedDeclaredDependencies>
                 <!-- auto-service isn't detected as use since the annotation has retention of source -->
-                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc4</unusedDeclaredDependency>
+                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc*</unusedDeclaredDependency>
```

Comment:
I actually meant a wildcard for the entire version, so we can change to whatever version comes later.

Reply:
Whoops!  Fixed in 67cb5999dff16e40430ff6d1dfa3bbcec11f25e